styleguide/screens/Patterns/HorizontalOverflowBar/HorizontalOverflowBar.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { applyContainerQuery } from 'react-container-query';
+import cx from 'classnames';
+import dedent from 'dedent';
+import uniqueId from 'lodash/fp/uniqueId';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, Note, C } from '../../../components/Scaffold/Scaffold';
+
+import { generateArray } from '../../../../utils/array/array';
+import HorizontalOverflowBar
+  from '../../../../components/Navigation/HorizontalOverflowBar/HorizontalOverflowBar';
+import horizontalOverflowBarCss
+  from '../../../../components/Navigation/HorizontalOverflowBar/HorizontalOverflowBar.css';
+
+import m from '../../../../globals/modifiers.css';
+
+const CenteredBar = applyContainerQuery(HorizontalOverflowBar, {
+  [horizontalOverflowBarCss.centered]: {
+    minWidth: 650,
+  },
+});
+
+const links = generateArray(4)
```

Comment:
Only generating four links here doesn't actually demonstrate what the component does. It needs to be set to a value which causes it to overflow the specimen's bounds, causing it be scrollable